indico/modules/events/timetable/templates/balloons/block.html

```diff
@@ -16,15 +16,15 @@
         <div class="title-header">
             <div class="title-caption">{% trans %}Block{% endtrans %}</div>
             {% if editable and can_manage_blocks %}
-                {% set title=' \'{}\''.format(block.title) if block.title else '' %}
+                {% set title="'%s'" | format(block.title) if block.title else '' %}
                 <div class="group right entry-action-buttons">
                     <a class="icon-edit i-button-icon js-dialog-action js-edit js-hide-balloon"
                        title="{% trans %}Edit session block{% endtrans %}"
-                       data-title="{% trans title=title %}Edit session block{{ title }}{% endtrans %}"></a>
+                       data-title="{% trans %}Edit session block{{ title }}{% endtrans %}"></a>
```

Comment:
missing space (either here or in the `{% set title %}`). I think having the space here makes more sense - the trailing space in case there is no title is not a big deal